apps/desktop/src/components/Settings/schemas.ts

```diff
@@ -96,6 +96,34 @@ export const getSections = (
                         controls: [{ kind: 'boolean', key: 'editor.wordWrap' }]
                     }
                 ]
+            },
+            {
+                id: 'spellcheck',
+                title: 'Spellcheck',
+                rows: [
+                    {
+                        id: 'spellcheck-enabled',
+                        label: 'Enable spellcheck',
+                        controls: [{ kind: 'boolean', key: 'editor.spellcheck.enabled' }]
+                    },
+                    {
+                        id: 'spellcheck-language',
+                        label: 'Language',
+                        controls: [{
+                            kind: 'select',
+                            key: 'editor.spellcheck.language',
+                            options: [
+                                { value: 'en-CA', label: 'English (Canada)' }
+                            ]
+                        }]
+                    },
+                    {
+                        id: 'spellcheck-dictionary',
+                        label: 'Personal Dictionary',
+                        description: 'Manage words added to your dictionary.',
+                        controls: [{ kind: 'collection', collectionId: 'personal-dictionary' }]
+                    }
```

Comment:
There is one oversight, an ignored list that the user can edit and remove words that are ignored if they choose to not ignore them later.

<img width="1405" height="1120" alt="Image" src="https://github.com/user-attachments/assets/69940e3a-c07c-47d9-803d-df55c9c3151d" />

The SPELLCHECK.md should be updated to mention this.